images/upload-gitlab-failure-logs/taxonomy.yaml

```diff
@@ -85,11 +85,15 @@ taxonomy:
 
     pod_not_found:
       grep_for:
-        - 'ERROR: Job failed \(system failure\): pods .+ not found'
+        - 'ERROR: Job failed \(system failure\):.+pods .+ not found'
 
     pod_failed:
       grep_for:
-        - 'ERROR: Job failed: pod .+ status is "Failed"'
+        - 'ERROR: Job failed.+:.+pod .+ status is ("Failed"|failed)'
+
+    helper_container_not_found:
+      grep_for:
+        - 'ERROR: Job failed.+:.+container helper not found in pod'
```

Comment:
Can you provide a link to an example of this?

Reply:
https://gitlab.spack.io/spack/spack/-/jobs/6253546